_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,40 @@
+## Node requirements
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later with [required dependencies](#kernel-dependencies). 
+  The following distributions have the required kernel, its dependencies, and are 
+  known to work well with {{site.prodname}} and {{include.orch}}.
+  - RedHat Linux 7{% if include.orch == "Kubernetes" or include.orch == "host protection" %}
+  - CentOS 7
+  - CoreOS Container Linux stable
+  - Ubuntu 16.04
+  - Debian 8
+  {% endif %}{% if include.orch == "OpenShift" %}
+  - CentOS 7
+  {% endif %}{% if include.orch == "OpenStack" %}
+  - Ubuntu 16.04
+  - CentOS 7
+  {% endif %}
+
+## Key/value store
+
+{{site.prodname}} {{page.version}} requires a key/value store accessible by all 
+{{site.prodname}} components. {% if include.orch == "Kubernetes" %} On Kubernetes,
+you can configure {{site.prodname}} to access an etcdv3 cluster directly or to
+use the Kubernetes API datastore.{% endif %}{% if include.orch == "OpenShift" %} On
+OpenShift, {{site.prodname}} can share an etcdv3 cluster with OpenShift, or
+you can set up an etcdv3 cluster dedicated to {{site.prodname}}.{% endif %}
+{% if include.orch == "OpenStack" %}If you don't already have an etcdv3 cluster
+to connect to, we provide instructions in the [installation documentation](./installation/).{% endif %}
+
+## Network requirements
```

Comment:
I think we should have a really brief introduction to this section, otherwise folks don't really know what they're reading.

Something like:


"Calico requires that the following types of traffic are not dropped by the network"

Also worth maybe another column where we can say for IPIP that it's only required when using IPIP mode.